meta-genivi-dev/meta-genivi-dev/recipes-extended/cannelloni/files/lanch_cannelloni.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Lanch cannelloni here.
+# Uncomment below line and provide valid configuration to start cannelloni at boot time
+
+#cannelloni -I slcan0 -R 127.0.0.1 -r 20000 -l 20000
```

Comment:
At this moment the configuration of Cannelloni is unknown. This is to be changed in the future. For now I had to only provide systemd service and empty lanch script.